### PR TITLE
Bug 1399625 - limit file size in main_summary

### PR DESCRIPTION
**Pull request by relud**
alternative solution with less change

### CODE REVIEW

File: src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala [hunk 1]
```diff
@@ -92,6 +92,8 @@ object MainSummaryView {
     val appVersion = opt[String]("version", descr = "Only process data from the given app version", required = false)
     val allHistograms = opt[Boolean]("all-histograms", descr = "Flag to use all histograms", required = false)
     val docType = opt[String]("doc-type", descr = "DocType of pings conforming to main ping schema", required=false, default=Some("main"))
+    // 500,000 rows yields ~ 200MB files in snappy+parquet
```
Comment (relud) [inline src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala]: this file `s3://telemetry-parquet/main_summary/v4/submission_date_s3=20180110/sample_id=0/` contains 1,677,177,577 bytes and 365,9217 rows. This yields ~ 483 bytes per row, and 457,893 rows for a 200MiB file. I rounded up to 500,000.

**Comment by relud** [comment]
@frank [based on this](https://github.com/apache/spark/blob/v2.2.0/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormatWriter.scala#L316-L324) I'm pretty sure it just writes records until it hits the limit.

**Comment by relud** [comment]
yeah, confirmed in my test run:

```
# aws s3 ls telemetry-test-bucket/main_summary/v4/submission_date_s3=20180121/sample_id=0/
2018-02-02 18:00:17  250952863 part-00005-a048c0c6-166b-4c40-a70b-05f75deaea18.c000.snappy.parquet
2018-02-02 18:00:17  232780737 part-00005-a048c0c6-166b-4c40-a70b-05f75deaea18.c001.snappy.parquet
2018-02-02 18:00:17  237617787 part-00005-a048c0c6-166b-4c40-a70b-05f75deaea18.c002.snappy.parquet
2018-02-02 18:00:17  242509592 part-00005-a048c0c6-166b-4c40-a70b-05f75deaea18.c003.snappy.parquet
2018-02-02 18:00:17  153382868 part-00005-a048c0c6-166b-4c40-a70b-05f75deaea18.c004.snappy.parquet
```

**Comment by fbertsch** [comment]
@relud in that case please run an example of this into `telemetry-test-bucket` so we can see what the results look like.

**Comment by fbertsch** [comment]
@relud given the simplicity of this change I think it's worth it anyways even with the small files that will be interspersed in the partitions.

**Comment by relud** [comment]
I was able to verify the output of DatasetComparator, and this didn't break anything for the test day

**Pull request merged**